### PR TITLE
Portfolio mobile menu

### PR DESCRIPTION
**Pull request by yodit93**
### In this branch, I have added:

- [ ] An event handler for the hamburger menu for displaying the mobile menu in response to the click event.
- [ ] An event listener for the close button for hiding the mobile menu options in response to the click event.
- [ ] An event handler for the mobile menu options for hiding the lists and for displaying the corresponding sections in response to click event.

By mistake I merged the two branches and now I created a new pull request. If it is needed to check the progress I have attached here the previous pull request link.

https://github.com/yodit93/My-portfolio/pull/13

### CODE REVIEW

File: styles/desktop.css [hunk 1]
```diff
@@ -7,22 +7,21 @@
...
+  .click-event {
+    display: inline-block;
   }
 
-  .icon-menu {
+  .close-btn {
     display: none;
   }
 
   .menu-lists {
```
Comment (christianbiring1) [inline styles/desktop.css:22]: - [ ] Kindly make sure you stick to the design as much as possible. spacing, color...
As you may notice on small screens there's a scroll which is prompting the user to scroll on the right to see the `hamburger` menu, Kinldy fix that.
`Hint: YOu may consider giving your nav-bar or body a width of 100%`

image

File: styles/desktop.css [hunk 2]
```diff
@@ -7,22 +7,21 @@
...
     display: flex;
+    flex-direction: row;
     list-style-type: none;
```
Comment (christianbiring1) [inline styles/desktop.css:25]: - [ ] Also make sure the popup layer fit the entire height of the screen even when the user ted to scroll all the way down. they don't need to see the content of the page.

image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @yodit93 , 
# STATUS: REQUEST CHANGES :recycle: 
soclose

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary` :+1: 
- [ ] good commits history :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .eslintrc.json:1]: **[Optional]**

- [ ] You are not using Github Flow which was requirement but you are using Gitflow. These two are two different things but in this project you are supposed to use [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md) 😊

**Review by HAMMAS-SALEEM — APPROVED** [review]
# Status: Approved :green_circle: :1st_place_medal: :100:
success

### Hey @yodit93, @christianbiring1, @chepkok3 👋

Great work on implementing the requested changes. Your project is complete. :blush:
There is nothing else to say about your project. :+1:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Requirements

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**